docker-compose/monitor/README.md

```diff
@@ -83,6 +85,40 @@ Then navigate to the Monitor tab at http://localhost:16686/monitor to view the R
 
 ![My Service RED Metrics](images/my_service_metrics.png)
 
+## Migrating to Span Metrics Connector 
```

Comment:
I intend to add something similar to this to the Jaeger documentation after this is approved.

Reply:
I would instead change the docs to only describe connector and point to readme for migration.